.vscode/launch.json

```diff
@@ -68,7 +68,8 @@
                 "--extensionDevelopmentPath=${workspaceFolder}/../vscode-cosmosdb",
                 "--extensionDevelopmentPath=${workspaceFolder}/../vscode-azurecontainerapps",
                 "--extensionDevelopmentPath=${workspaceFolder}/../vscode-azurespringcloud",
-                "--extensionDevelopmentPath=${workspaceFolder}/../vscode-azurelogicapps"
+                "--extensionDevelopmentPath=${workspaceFolder}/../vscode-azurelogicapps",
+                "--extensionDevelopmentPath=${workspaceFolder}/../vscode-azurewebpubsub",
```

Comment:
This isn't required and can be removed. This is meant for our own testing of Resource Groups. If you want to test your extension with Resource Groups, add RGs to the launch.json of your extension.